requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==1.8.2
+virtualenv==1.7.2
```

Comment:
I wouldn't put virtualenv in the requirements file... it's more of a developer machine set up thing than a project dependancy.